corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -81,6 +82,43 @@ def start_reindex(self, cname, reindex_batch_size=1000, requests_per_second=None
             + f"\"grep '{task_number}.*ReindexResponse' /opt/data/elasticsearch*/logs/*.log\""
             + "\n\n")
 
+    def reindex_partial_index(self, cname, domain):
+        adapter = doc_adapter_from_cname(cname)
+        if not adapter.parent_index_name:
+            raise IndexNotSubindexException(f"Adapter for {cname} is not a partial index")
+
+        source_index = adapter.parent_index_name
+        destination_index = adapter.primary.index_name
+
+        logger.info(f"Preparing index {destination_index} for reindex")
+        self._prepare_index_for_reindex(destination_index)
+
+        logger.info("Starting partial reindex process")
+        task_id = es_manager.reindex(source_index, destination_index, query={"term": domain})
+
+        logger.info(f"Copying docs from index {source_index} to index {destination_index} for {domain}")
+        task_number = task_id.split(':')[1]
+        print("\n\n\n")
+        logger.info("-----------------IMPORTANT-----------------")
+        logger.info(f"TASK NUMBER - {task_number}")
+        logger.info("-------------------------------------------")
+        logger.info("Save this Task Number, You will need it later for verifying your partial reindex process")
+        print("\n\n\n")
+        # This would display progress untill reindex process is completed
+        check_task_progress(task_id)
+
+        print("\n\n")
+
+        self.display_source_destination_doc_count(adapter)
```

Comment:
This needs to be updated since this method doesn't handle a partial index adapter. Is it the case that when reindexing to a partial index, will we want to cleanup tombstones on the partial index? That is what currently happens when calling this display_source_destination_doc_count (which feels like a surprising side effect?)

Reply:
I updated this in 55d17c38142570c81b191c6ea834030d9e48256c, but left the part that cleans up tombstones in.